### PR TITLE
Prepare plugin for release

### LINKED ISSUE #13: Prepare the plugin for publish on NPM & gatsby
The plugin should be prepared to be released on NPM: 
- Script checking releases `check-release.sh`
- Github Action Publish

But also to be visible on Gatsby's plugin list. See: 
- https://www.gatsbyjs.com/docs/how-to/plugins-and-themes/submit-to-plugin-library/ 
- https://www.gatsbyjs.com/docs/files-gatsby-looks-for-in-a-plugin/

### PR DESCRIPTION
**Pull request by mdubus**
# Pull Request

## What does this PR do?
Closes #13

## PR checklist
Please check if your PR fulfills the following requirements:
- [x] Does this PR fix an existing issue?
- [x] Have you read the contributing guidelines?
- [x] Have you made sure that the title is accurate and descriptive of the changes?

## What's inside this PR ? 
- Addition of the `check-release.sh` script
- Addition of a new check making the tests fail with Node 16

I also checked that the following elements are present for the release: 
- .github/workflows/release-drafter.yml
- .github/release-draft-template.yml
- .github/workflows/publish.yml
- `keywords` inside the `package.json` following the [Gatsby Guidelines](https://www.gatsbyjs.com/docs/how-to/plugins-and-themes/submit-to-plugin-library/)
- Update of the Readme

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -131,6 +130,23 @@ The plugin accepts the following options for further customization :
...
+
+(This plugin may work with the older Gatsby versions, but these are not tested nor officially supported at this time.)
+
+**Supported MeiliSearch versions**:
+
+This package only guarantees the compatibility with the [version v0.24.0 of MeiliSearch](https://github.com/meilisearch/MeiliSearch/releases/tag/v0.24.0).
+
+**Node / NPM versions**:
+
+- NodeJS >= 12.10 <= 14
```
Comment (bidoubiwa) [inline README.md:145]: Node 16 does not work ?

Comment (mdubus) [reply README.md]: It does ;) Just edited it, thanks 🙏

File: .github/workflows/tests.yml [hunk 2]
```diff
@@ -28,7 +28,43 @@ jobs:
         run: yarn --dev
       - name: Run style check
         run: yarn lint
-  tests-run:
+  tests:
```
Comment (curquiza) [inline .github/workflows/tests.yml:31]: Don't forget to add those new tests in bors.toml

File: .github/scripts/check-release.sh [hunk 3]
```diff
@@ -0,0 +1,13 @@
+#!/bin/sh
```
Comment (curquiza) [inline .github/scripts/check-release.sh:1]: where this script is used?

**Review by curquiza — COMMENTED** [review]
Sorry if the PR is not finished yet 😇 just in case you forgot it

File: .github/workflows/tests.yml (hunk 2 above)
Comment (mdubus) [reply .github/workflows/tests.yml]: I completely forgot 🙈 Thanks a lot !

File: .github/scripts/check-release.sh (hunk 3 above)
Comment (mdubus) [reply .github/scripts/check-release.sh:1]: Also forgot this one ... Just added a `publish.yml` file. Thanks a lot !

File: .github/workflows/publish.yml [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+          node-version: 14
+          registry-url: https://registry.npmjs.org/
+      - name: Check release validity
+        run: sh .github/scripts/check-release.sh
+      - name: Install dependencies
+        run: yarn install
+      - name: Publish
+        run: npm publish .
+        env:
+          NODE_AUTH_TOKEN: ${{secrets.NPM_TOKEN}}
```
Comment (curquiza) [inline .github/workflows/publish.yml:22]: FYI, I've just added the `NPM_TOKEN` secret to this repo

Capture d’écran 2021-12-09 à 19 00 22

File: bors.toml [hunk 5]
```diff
@@ -1,5 +1,7 @@
 status = [
-  'linter_check'
+  'linter_check',
+  'tests-run',
```
Comment (curquiza) [inline bors.toml:3]: You have 2 tests running, `tests-run (14)` and `tests-run (16)`

Capture d’écran 2021-12-09 à 19 01 35

File: .github/workflows/publish.yml (hunk 4 above)
Comment (mdubus) [reply .github/workflows/publish.yml:22]: Thank you so much ❤️

File: bors.toml (hunk 5 above)
Comment (mdubus) [reply bors.toml:3]: I know, I'm trying to understand where this one comes from 🧐. If you have any idea...

Comment (curquiza) [reply bors.toml]: They come from the CI you created, it's normal, you just need to update bors.toml accordingly

**Review by curquiza — APPROVED** [review]
💪

**Review by bidoubiwa — APPROVED** [review]
LGTM 🔥 Fantastic work

**Pull request merged**